packages/builder/src/components/start/ChooseIconModal.svelte

```diff
@@ -0,0 +1,125 @@
+<script>
+  import { ModalContent, Modal, Icon, ColorPicker, Label } from "@budibase/bbui"
+  import { apps } from "stores/portal"
+
+  export let app
+  let modal
+  $: selectedIcon = app?.icon?.name
+  $: selectedColor = app?.icon?.color
+
+  let iconsList = [
+    "Actions",
+    "ConversionFunnel",
+    "App",
+    "Briefcase",
+    "Money",
+    "ShoppingCart",
+    "Form",
+    "Help",
+    "Monitoring",
+    "Sandbox",
+    "Project",
+    "Organisations",
+    "Magnify",
+    "Launch",
+    "Car",
+    "Camera",
+    "Bug",
+    "Channel",
+    "Calculator",
+    "Calendar",
+    "GraphDonut",
+    "GraphBarHorizontal",
+    "Demographic",
+    "Apps",
+  ]
+  export const show = () => {
+    modal.show()
+  }
+  export const hide = () => {
+    modal.hide()
+  }
+
+  const onCancel = () => {
+    selectedIcon = ""
+    selectedColor = ""
+    hide()
+  }
+
+  const changeColor = val => {
+    selectedColor = val
+  }
+
+  const save = async () => {
+    await apps.updateIcon(app.instance._id, {
+      name: selectedIcon,
+      color: selectedColor,
+    })
+  }
+</script>
+
+<Modal bind:this={modal} on:hide={onCancel}>
+  <ModalContent
+    title={"Edit Icon"}
+    confirmText={"Save"}
+    onConfirm={() => save()}
+  >
+    <div class="scrollable-icons">
+      <div class="title-spacing">
+        <Label>Select an Icon:</Label>
+      </div>
+      <div class="grid">
+        {#each iconsList as item}
+          <div
+            class="icon-item"
+            style="color: {item === selectedIcon ? selectedColor : ''}"
+            on:click={() => (selectedIcon = item)}
+          >
+            <Icon name={item} />
+          </div>
+        {/each}
+      </div>
+    </div>
+    <div class="color-selection">
+      <div>
+        <Label>Select a Color:</Label>
```

Comment:
NAB: same as above, would just case this as "Select a color" (without the colon as well)